src/confd/src/infix-system-software.c

```diff
@@ -51,12 +50,49 @@ static int infix_system_sw_install(sr_session_ctx_t *session, uint32_t sub_id,
 	return srerr;
 }
 
+static int infix_system_sw_set_boot_order(sr_session_ctx_t *session, uint32_t sub_id,
+					  const char *path, const sr_val_t *input,
+					  const size_t input_cnt, sr_event_t event,
+					  unsigned request_id, sr_val_t **output,
+					  size_t *output_cnt, void *priv)
+
+{
+
+        /* primary secondary net, longest allowed boot-order */
+	char boot_order[23] = "";
+	for (size_t i = 0; i < input_cnt; i++) {
+		 if (i != 0)
+			 strlcat(boot_order, " ", sizeof(boot_order));
+		 const sr_val_t *val = &input[i];
+		 strlcat(boot_order, val->data.string_val, sizeof(boot_order));
+	 }
+
+	 if (fexist("/sys/firmware/devicetree/base/chosen/u-boot,version")) {
+		 if (systemf("fw_setenv BOOT_ORDER %s", boot_order)) {
+			 ERROR("Set-boot-order: Failed to set bootorder in U-boot");
```

Comment:
Boot order is two words both in spoken English and in our model.  Also, maybe use "Das U-Boot" or "U-Boot", capital letter(s) since it's a name.